esmf_regrid/schemes.py

```diff
@@ -18,39 +18,43 @@
 ]
 
 
-def _bounds_cf_to_simple_1d(cf_bounds):
-    assert (cf_bounds[1:, 0] == cf_bounds[:-1, 1]).all()
-    simple_bounds = np.empty((cf_bounds.shape[0] + 1,), dtype=np.float64)
-    simple_bounds[:-1] = cf_bounds[:, 0]
-    simple_bounds[-1] = cf_bounds[-1, 1]
-    return simple_bounds
-
-
 def _cube_to_GridInfo(cube):
     # This is a simplified version of an equivalent function/method in PR #26.
     # It is anticipated that this function will be replaced by the one in PR #26.
     #
     # Returns a GridInfo object describing the horizontal grid of the cube.
     # This may be inherited from code written for the rectilinear regridding scheme.
-    lon = cube.coord("longitude")
-    lat = cube.coord("latitude")
-    # Ensure coords come from a proper grid.
-    assert isinstance(lon, iris.coords.DimCoord)
-    assert isinstance(lat, iris.coords.DimCoord)
-    # TODO: accommodate other x/y coords.
-    # TODO: perform checks on lat/lon.
+    lon, lat = cube.coord(axis="x"), cube.coord(axis="y")
     #  Checks may cover units, coord systems (e.g. rotated pole), contiguous bounds.
     if cube.coord_system() is None:
         crs = None
     else:
         crs = cube.coord_system().as_cartopy_crs()
+    londim, latdim = len(lon.points.shape), len(lat.points.shape)
```

Comment:
This is operating on the `points` property, above when doing these checks we were operating on the object itself. Is this because we now have iris objects instead of numpy arrays?

Reply:
I believe thats correct, here `lon` represents a coord whereas in `_esmf_sdo`, `lons` represented an array.